src/geocoding/location.py

```diff
@@ -24,7 +25,8 @@ def geometry_value(self) -> Optional[str]:
         return self._location.get(_VALUE_ATTR_NAME, None)
 
     def is_geojson(self):
-        return self.geometry_type() == _GEOJSON_TYPE
+        return self.geometry_type() == _GEOJSON_TYPE or \
+               self.geometry_type() == _GEOJSON_LD_TYPE
```

Comment:
to support proper management also of `GeoProperty` if NGSI-LD likey payload is sent